File: HTTPShortcuts/app/src/main/java/ch/rmy/android/http_shortcuts/utils/Connectivity.java
package ch.rmy.android.http_shortcuts.utils;

import android.content.Context;
import android.net.ConnectivityManager;

public class Connectivity {

    public static boolean isNetworkConnected(Context context) {
        ConnectivityManager connectivityManager = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
        return connectivityManager.getActiveNetworkInfo() != null;
    }

}

File: HTTPShortcuts/app/src/main/java/ch/rmy/android/http_shortcuts/utils/Connectivity.kt
package ch.rmy.android.http_shortcuts.utils

import android.content.Context
import android.net.ConnectivityManager

object Connectivity {

    fun isNetworkConnected(context: Context): Boolean {
        val connectivityManager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
        return connectivityManager.activeNetworkInfo != null
    }

}
